docs/src/main/asciidoc/spring-di.adoc

```diff
@@ -261,87 +389,132 @@ public class GreetingResourceTest {
 
-|@Repository
-|@Singleton
+|`@Repository`
+|`@Singleton`
 |By default Spring stereotype annotations are singleton beans
 
-|@Configuration
-|@ApplicationScoped
-|In CDI a producer bean isn't limited to the application scope, it could just as well be @Singleton or @Dependent
+|`@Configuration`
+|`@ApplicationScoped`
+|In CDI a producer bean is not limited to the application scope, it could just as well be `@Singleton` or `@Dependent`
 
-|@Bean
-|@Produces
+|`@Bean`
+|`@Produces`
 |
 
-|@Scope
+|`@Scope`
 |
-|Doesn't have a one-to-one mapping to a CDI annotation. Depending on the value of @Scope, one of the @Singleton, @ApplicationScoped, @SessionScoped, @RequestScoped, @Dependent could be used
+|Does not have a one-to-one mapping to a CDI annotation. Depending on the value of `@Scope`, you can use `@Singleton`, `@ApplicationScoped`, `@SessionScoped`, `@RequestScoped`, or `@Dependent`
 
-|@ComponentScan
+|`@ComponentScan`
 |
-|Doesn't have a one-to-one mapping to a CDI annotation. It is not used in Quarkus because Quarkus does all classpath scanning at build time.
+|Does not have a one-to-one mapping to a CDI annotation. It is not used in {BrandName} because {BrandName} does all classpath scanning at build time.
 
-|@Import
+|`@Import`
 |
-|Doesn't have a one-to-one mapping to a CDI annotation.
+|Does not have a one-to-one mapping to a CDI annotation.
 |===
 
+ifdef::upstream[]
 == More Spring guides
 
-Quarkus has more Spring compatibility features. See the following guides for more details:
-
-* link:spring-web[Quarkus - Extension for Spring Web]
-* link:spring-data-jpa[Quarkus - Extension for Spring Data JPA]
-* link:spring-data-rest[Quarkus - Extension for Spring Data REST]
-* link:spring-security[Quarkus - Extension for Spring Security]
-* link:spring-cloud-config-client[Quarkus - Reading properties from Spring Cloud Config Server]
-* link:spring-boot-properties[Quarkus - Extension for Spring Boot properties]
-* link:spring-cache[Quarkus - Extension for Spring Cache]
-* link:spring-scheduled[Quarkus - Extension for Spring Scheduled]
+{BrandName} provides more Spring compatibility features.
+See the following guides for details:
+
+* link:spring-web[{BrandName} - Extension for Spring Web]
+* link:spring-data-jpa[{BrandName} - Extension for Spring Data JPA]
+* link:spring-data-rest[{BrandName} - Extension for Spring Data REST]
+* link:spring-security[{BrandName} - Extension for Spring Security]
+* link:spring-cloud-config-client[{BrandName} - Reading properties from Spring Cloud Config Server]
+* link:spring-boot-properties[{BrandName} - Extension for Spring Boot properties]
+* link:spring-cache[{BrandName} - Extension for Spring Cache]
+* link:spring-scheduled[{BrandName} - Extension for Spring Scheduled]
+endif::[]
+
+ifdef::downstream[]
+[role="_additional-resources"]
+.Additional resources
+* link:{NATIVE_EXECUTABLE_QUARKUS}[Compiling and starting your {BrandName} application as a native executable.]
```

Comment:
Could this link not apply to community docs as well? I think the "Additional resources" section is something that could benefit community docs.

Also, if you wanted to get fancy `[role="_additional-resources"]` is easier to insert via script because `.Additional resources` acts as a marker. Like the abstract role, though, it is pretty harmless for community docs and just gets ignored during processing. At least that's the case for Infinispan docs.

Reply:
@oraNod I see your point and I agree that it could benefit the upstream version of the guide, too: However, consider that according to PV2 compatibility requirements, the additional resources section has some limitations on how it must be written to be publishable in Pantheon 2 (As far as I know, at least). 
Things such as:
* no blank lines between the section title and the list of links in the section
* no introductory paragraph or any other text except for an unordered list of links

It seems too restrictive a  changes to force on contributors.
Besides, most of the guides that are available in the upstream have not been made available downstream yet.
I don't think I can reconcile all of the above under one section that would fit both upstream and downstream.

Reply:
Right. I guess I missed the entire `== More Spring guides` section for community. I'd probably leave it as-is then.